crates/ruma-events/src/call.rs

```diff
@@ -6,6 +6,7 @@ pub mod answer;
 pub mod candidates;
 pub mod hangup;
 pub mod invite;
+pub mod member;
```

Comment:
This needs to be behind `#[cfg(feature = "msc3401")]` (and that feature has to be added to `ruma-events`' and `ruma`s `Cargo.toml`.